### ISSUE TITLE
How can this be used with a blueprint mod?

### ISSUE THREAD

**Issue by unstoppablecarl** [simonlourson/blueprintnotincluded#2]
Is there a way to use the blueprint json files in the game with a mod?

**Comment by nkv123** [comment]
yes there is, upload created blueprint as json if it fails try as binary. Then edit, save and download respectivly.
